storage/kafkasql/src/main/java/io/apicurio/registry/storage/impl/kafkasql/KafkaSqlUpgrader.java

```diff
@@ -29,26 +30,27 @@
 @ApplicationScoped
 public class KafkaSqlUpgrader {
```

Comment:
Realizing that this probably contributes to slow startup times for kafkasql, eh?  Perhaps for 3.0 we should consider excluding hashes and similar from the Kafka messages and generate them during message consumption.  I'm not sure if that's feasible or not, but it would mean we wouldn't need to run these upgraders.

Reply:
Note: this has nothing to do with your PR, just a thought I had while reviewing.  :)